docs/js/doc-links.yml

```diff
@@ -117,7 +117,9 @@
           items:
             - title: Introduction to Service Preview and Edge Control
               link: /docs/latest/topics/using/edgectl/
-            - title: Edge Control
+            - title: Service Preview in Action
```

Comment:
This new page was extracted from the old "Edge Control" page, and "Edge Control" was renamed "Edge Control Reference".

The first, "Service Preview in Action" is intended to be a quickstart tutorial -- but needs love.
The second, "Edge Control Reference" is intended to be the `edgectl` cmd reference.